scripts/building/check_enforcer.py

```diff
@@ -0,0 +1,283 @@
+# Import necessary libraries
+import os
+import fnmatch
+import subprocess
+import time
+import argparse
+import json
+import sys
+
+# Define variables
+github_repository = "microsoft/promptflow"
+snippet_debug = 1  # Write debug info to console.
+merge_commit = ""
+loop_times = 30
+github_workspace = os.path.expanduser("~/promptflow/")
+
+# Special cases for pipelines that need to be triggered more or less than default value 1.
+# If 0, the pipeline will not be ignored in check enforcer.
+special_care = {"sdk_cli_tests": 4, "sdk_cli_azure_test": 4}
```

Comment:
this might be useful in some scenarios, maybe it will be better to add an example workflow as demonstration and comment it, so that everyone can know what & how to do this

Reply:
also, I think it's better to point out what's the key... I just get confused on what should I pick...